### ISSUE TITLE
Import/Export followers list

### ISSUE THREAD

**Issue by StarrWulfe** [snarfed/bridgy-fed#355]
All mastodon compatible clients have a way to import and export followers from your follow list should you want to take your account elsewhere. BridgyFed should iterate and offer this functionality. 

# for reference:

[Moving or leaving accounts ](https://docs.joinmastodon.org/user/moving/)

**Comment by snarfed** [comment]
Sure! I'm happy to do this manually for anyone who wants it.

**Comment by snarfed** [comment]
(I don't plan to prioritize automated import/export myself soon, but PRs are always welcome. I'll leave this issue open to track!)

**Comment by StarrWulfe** [comment]
The above could include OPML as an import/export format as well.

**Comment by snarfed** [comment]
True! First step in implementing would be an OPML feature request in https://granary.io/ .

**Comment by tmichellemoore** [comment]
I would vote for this also. 😊
